### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by c3r**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Bash Infinity
+Bash Infinity Yooo
```
Comment (niieani) [inline README.md]: spoko

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 =============
```
Comment (niieani) [inline README.md]: brakuje fajnego Yoo

Comment (c3r) [inline README.md]: ok

**Comment by morenoh149** [comment]
🤐

**Comment by niieani** [comment]
@morenoh149 sorry about that, we were just testing GitHub PR reviewing! Locking the thread.

**Pull request closed without merging**